google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/server/instance/AppEngineServerModel.java

```diff
@@ -420,6 +432,9 @@ public void setAutomaticRestart(Boolean automaticRestart) {
 
   @Override
   public String getDevAppserverLogLevel() {
+    if (StringUtil.isEmpty(settings.getDevAppserverLogLevel())) {
```

Comment:
The user can't set the value to null for log level. This is for the case where no state has been serialized and we want to set an initial value..